app/models/resource.py

```diff
@@ -131,7 +131,13 @@ def print_resources():
             print resource.text_descriptors
             print resource.option_descriptors
 
-
+    @staticmethod
+    def get_resources():
```

Comment:
@weronica that's my bad..i made this method before: i'll remove it in my next PR